profiling/variorum-connector/variorum-connector.cpp

```diff
@@ -357,25 +509,102 @@ void kokkosp_init_library(const int loadSeq, const uint64_t interfaceVer,
 }
 
 void kokkosp_finalize_library() {
-  if (usingMPI) {
-    variorum_call_mpi();
-  } else {
-    variorum_call();
+  /*	std::cout << "kokkos library call\n" << std::endl;
+    if (usingMPI) {
+      variorum_call_mpi();
+    } else {
+     variorum_call();
+    }
+    time_t total_time;
+    time_t end_time;
+    time(&end_time);
+    std::cout << "End Time: " << end_time << "\nStart Time: " << start_time
+              << "\n";
+    total_time = end_time - start_time;
+
+    std::cout << "The kokkos library was alive for " << total_time << "
+    seconds."
+              << std::endl;*/
```

Comment:
```suggestion
```

Reply:
@twilk10  Is this suggestion OK with you? If so, can you commit it? 

It is good with me.